app/javascript/shared/utils/FileUtils.re

```diff
@@ -12,12 +13,26 @@ let isImage = file => {
   };
 };
 
-let isValid = (~maxSize=defaultMaxSize, ~image=false, file) => {
-  let sizeValid = hasValidSize(~maxSize, file);
+let isVideo = file => {
+  switch (file##_type) {
+  | "video/mp4"
+  | "video/mov"
+  | "video/wmv"
+  | "video/avi"
+  | "video/flv"
+  | _ => false
+  };
+};
+
+let isValid = (~maxSize=defaultMaxSize, ~image=false, ~video=false, file) => {
+  let maxSize = video === true ? defaultVideoMaxSize : maxSize
```

Comment:
In Reason, `==` is actually the preferred comparison operator.

- `==` means value equality (deep value comparison)
- `===` means referential equality (must be at same memory address).

Reply:
🤦